docs/arch_and_scale/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _RabbitMQ Clustering Guide: https://www.rabbitmq.com/clustering.html
+
+Architecture and Scaling
+========================
+
+Pulp's architecture has five components of it. Each of these can be horizontally scaled
+independently for both high availability and/or additional capacity for that part of the
+architecture.
+
+WSGI application
+  Pulp's web application is served by one or more WSGI webservers. See the
+  :ref:`wsgi-application` docs for more info on deploying and scaling this component.
+
+Workers
+  Pulp's tasking system requires at least one running worker. Additional workers can be added to add
+  capacity to the tasking system.
+
+Resource Manager
+  Pulp's tasking system requires at least one running resource manager. Although its required for
```

Comment:
"Although it's"--missing the apostrophe in the contraction.

Reply:
ty, fixed in next push.